src/features/components/templates/navigation/template/desktop-menu/styles.scss

```diff
@@ -44,6 +44,11 @@
         padding: 2.4rem 0.8rem;
         animation-duration: 250ms;
         animation-timing-function: ease;
+        @media screen and (max-height: 700px) {
+            block-size: 100vh;
+            padding-block-end: 10rem;
+            overflow: scroll;
+        }
```

Comment:
Can we add this part under the **trade_active** class? Because the block-size increases for the other tabs too.

```suggestion
        @media screen and (max-height: 820px) {
            block-size: 90vh;
            overflow: scroll;
        }
```

Reply:
thanks man for pointing this out, adding it to `.trade_active` will cause other issues. 
fixed it in other way. 
thanks :pray: